### ISSUE TITLE
Add instructions how to add a new model

### ISSUE THREAD

**Issue by schillic** [JuliaReach/ReachabilityBenchmarks#46]

**Comment by mforets** [comment]
What did we want to do here, more exactly? Some general instructions on adding a model to ReachabilityBenchmarks?

**Comment by schillic** [comment]
I think just explain how to construct a model (both continuous and hybrid). Maybe with the option to define a matrix manually and with loading a `.mat` file.
(We could just copy an existing model and add some comments.)

**Comment by mforets** [comment]
Shall we add this as a section in the Reachability documentation? I think that we can split between linear / nonlinear and continuous / hybrid.

**Comment by schillic** [comment]
Yes, I think we can do that and then link to that part of the manual.
